### PR TITLE
test(extension): Switching to less pools modal tests

### PR DESCRIPTION
**Pull request by oldGreg5**
[Jira LW-9642](https://input-output.atlassian.net/browse/LW-9642)

Switching to less pools modal tests

### CODE REVIEW

File: packages/e2e-tests/src/assert/switchingPoolsModalAssert.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { expect } from 'chai';
+import { t } from '../utils/translationService';
+import SwitchingStakePoolModal from '../elements/multidelegation/SwitchingStakePoolModal';
+
+class SwitchingPoolsModalAssert {
+  async assertSeeSwitchingToLessPoolsModal() {
+    await SwitchingStakePoolModal.title.waitForDisplayed();
+    expect(await SwitchingStakePoolModal.title.getText()).to.equal(await t('modals.poolsManagement.title', 'staking'));
```
Comment (wklos-iohk) [inline packages/e2e-tests/src/assert/switchingPoolsModalAssert.ts:8]: Would you mind trying to use `await t(...)` without 2nd parameter? It seems to be no longer needed

File: packages/e2e-tests/src/features/MultidelegationDelegatedFundsMultiplePoolsExtended.feature [hunk 2]
```diff
@@ -180,3 +180,23 @@ Feature: Staking Page - Delegated funds - Multiple pools - Extended View
     When I press keyboard Escape button
     # When I press keyboard Enter button
     Then Drawer is not displayed
+
+  @LW-10998 @Testnet
```
Comment (wklos-iohk) [inline packages/e2e-tests/src/features/MultidelegationDelegatedFundsMultiplePoolsExtended.feature:184]: `@Testnet` tag is already provided at the to of the file

**Comment by pczeglik-iohk** [comment]
# Allure Report
[`allure-report-publisher`](https://github.com/andrcuns/allure-report-publisher) generated test report!



**smokeTests**: ✅ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/smoke/linux/chrome/6293/10005177756/index.html) for [f86d09ae](https://github.com/input-output-hk/lace/pull/1306/commits/f86d09aecb8931dff681a788be518ecdee39a1e3)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 32     | 0      | 0       | 0     | 32    | ✅     |

File: packages/e2e-tests/src/assert/switchingPoolsModalAssert.ts (hunk 1 above)
Comment (oldGreg5) [reply packages/e2e-tests/src/assert/switchingPoolsModalAssert.ts]: Good point, that's new thanks

**Review by wklos-iohk — APPROVED** [review]

**Pull request merged**